agent/agent.go

```diff
@@ -966,24 +976,25 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string,
 func (a *Agent) endTaskResponse(ctx context.Context, tc *taskContext, status string, message string) *apimodels.TaskEndDetail {
```

Comment:
I'm not sure what our convention is but don't use the agent object anymore- so we could remove it off of the agent struct.